src/pages/Tournament/TournamentList/parts.tsx

```diff
@@ -0,0 +1,147 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import sortedUniqBy from 'lodash/sortedUniqBy';
+
+import { Table, Button } from 'antd';
+
+import { PageTitle, Row } from 'components/common';
+import { useUserContext } from 'components/UserContext';
+import { TournamentPreview } from 'utils/apiResponseShapes';
+import { TournamentTypeDisplays } from './displays';
+import CreateTournamentDialog from './CreateTournamentDialog';
+import { useHistory, useRouteMatch } from 'react-router-dom';
```

Comment:
Codacy has a fix for the issue: [Replace `useHistory,·useRouteMatch·}·from·'react-router-dom` with `TournamentTypeDisplays·}·from·'./displays`](https://app.codacy.com/gh/othello-organisation-singapore/othello-storm-system-frontend/pullRequest?prid=8062249)
```suggestion
import { TournamentTypeDisplays } from './displays';
```